frigate/app.py

```diff
@@ -337,6 +342,13 @@ def cleanup_timeline_db(db: SqliteExtDatabase) -> None:
 
         migrate_db.close()
 
+    def init_embeddings(self) -> None:
+        self.embeddings: Optional[Embeddings] = None
```

Comment:
For all of these new services I think it would be good to only run them if this has been enabled in the config.

Reply:
That is the case with the client. The __init__ method will return a ValueError if `semantic_search` is not enabled and `self.embeddings` will remain `None`. See: https://github.com/blakeblackshear/frigate/pull/8998/files#diff-a87fb799a0f5537bd6fb9049628a14131fb21e8bc2f0fc21363d664bb6aff7feR50-R51